test/e2e-runner/src/windows/scenarios/installation.rs

```diff
@@ -0,0 +1,56 @@
+use std::thread;
+use std::time::Duration;
+
+use crate::tools::config;
+use crate::tools::logs::show_logs;
+use crate::tools::test::TestResult;
+use crate::windows;
+use tracing::info;
+
+const DEFAULT_STATUS_PORT: u16 = 51200;
+const SERVICE_NAME: &str = "newrelic-agent-control";
+
+#[derive(Debug, clap::Parser)]
+pub struct Args {
+    /// Path to the Windows zip package file
+    #[arg(short, long)]
+    zip_package: String,
+}
+
+/// Runs a complete Windows E2E installation test.
+pub fn test_installation(args: Args) -> TestResult<()> {
+    let zip_package = &args.zip_package;
+    info!("Installing agent-control...");
+    let output = windows::install::install_agent_control(zip_package, true)?;
+    info!("Installation completed successfully. Showing installation output");
+    println!("---\n{output}\n---");
```

Comment:
Nit: Thinking about having more tests, we might want these logs inside the install function. Same with the one above the function.

Reply:
Yeah, it probably makes sense

Reply:
Changed here: [f76b7ac](https://github.com/newrelic/newrelic-agent-control/pull/1965/commits/f76b7ac536fe641eb466628bdccf9997b09c3f03)